libs/sdk-backend-tiger/src/backend/workspace/insights/index.ts

```diff
@@ -103,23 +98,37 @@ export class TigerWorkspaceInsights implements IWorkspaceInsightsService {
                 });
         });
 
-        // tiger does not support the "updated" property of the metadata objects at the moment
-        // -> fall back to title ordering in a future-compatible way if "updated" ordering was requested
+        // tiger does not support ordering by "modifiedAt,createdAt" properties of the metadata objects at
+        // the moment: sort the insights manually here (by date desc, by title asc if the date is the same)
         let sanitizedOrder = allInsights;
-        if (usesOrderingByUpdated && allInsights.length > 0) {
-            // tiger started sending "updated" property -> use it to sort
-            if (insightUpdated(allInsights[0])) {
-                sanitizedOrder = sortBy(allInsights, (insight) => insightUpdated(insight));
-            }
-            // tiger still does not support the "updated" property -> sort by title
-            else {
-                sanitizedOrder = sortBy(allInsights, (insight) => insightTitle(insight).toUpperCase());
-            }
+        if (requestParameters.sort === undefined && allInsights.length > 0) {
+            sanitizedOrder = [...allInsights].sort((insightA, insightB) => {
+                const insightDateA = insightUpdated(insightA) ?? insightCreated(insightA) ?? "";
+                const insightDateB = insightUpdated(insightB) ?? insightCreated(insightB) ?? "";
+                const insightTitleA = insightTitle(insightA);
+                const insightTitleB = insightTitle(insightB);
+                return insightDateB.localeCompare(insightDateA) || insightTitleA.localeCompare(insightTitleB);
```

Comment:
This is technically a change as before we were comparing with `.toUpperCase()` but I think it's fine how it's now, cannot recall what was the motivation for that previously, so keep it as you have it now.

Reply:
Fixed.